opel_omega_2001.dbc

```diff
@@ -0,0 +1,69 @@
+VERSION ""
+
+NS_:
+BS_:
+BU_:
+
+BO_ 272 TCU_Data1: 8 TCU
+    SG_ TorqueRequest1   : 15| 8@0+ (1.0,0.0) [0.0|255] ""  ABS,ESP,ECU
+    SG_ TorqueRequest2   : 31| 8@0+ (1.0,0.0) [0.0|255] ""  ABS,ESP,ECU
+    SG_ OutputShaftSpeed : 55|16@0+ (1.0,0.0) [0.0|65535] "rpm"  ABS,ESP,ECU
+
+BO_ 288 ESP_Data1: 8 ESP
+    SG_ ABD_Active        :  4|1@0+ (1.0,0.0) [0.0|1.0] "yes/no"  ECU,TCU
+    SG_ TorqueRequestFast : 15|8@0+ (1.0,0.0) [0.0|255] ""  ECU,TCU
+    SG_ TorqueRequestSlow : 31|8@0+ (1.0,0.0) [0.0|255] ""  ECU,TCU
+
+BO_ 384 SAS_Data: 8 SAS
+    SG_ SteeringAngle :  0|16@1- (0.1,0.0) [-3276.8|3276.7] "yes/no"  ECU,TCU
+    SG_ SteeringSpeed : 16| 8@1+ (1.0,0.0) [0.0|255] ""  ECU,TCU
+
+BO_ 416 ECU_Data1: 8 ECU
+    SG_ RPM            : 15|16@0+ (1.0,0.0) [0.0|65535] "rpm"  ABS,ESP,TCU
```

Comment:
two byte field not aligned with bytes? "15" is start in bits, right?

Reply:
It is aligned. But it uses Motorola byte sequence.
You can check it in your Vector tool.
P.S. Yesterday I had eaten the heap of shit when I was creating this database with such byte sequence. It looks weird.(

Reply:
![image](https://user-images.githubusercontent.com/112559899/187939172-b0199380-95a5-4ef4-af86-debfb92786c3.png)
It looks so in Vector CANdb++. :)

Reply:
Wow, today I've learned! Thank you.

Reply:
> Wow, today I've learned! Thank you.

So probably it's time to push MR further? Who does it usually here?)

Reply:
Definitely not me, I am with the same board with you waiting for mine to be pushed :)

I guess that opendbc for non-autopilot cases is understandably not on top of the priority for maintainers?

Reply:
> Definitely not me, I am with the same board with you waiting for mine to be pushed :)
> 
> I guess that opendbc for non-autopilot cases is understandably not on top of the priority for maintainers?

R U that guy who usually fights with Maxi on Turbobazar forum? :)